ArduCopter/Parameters.cpp

```diff
@@ -1279,6 +1279,9 @@ void Copter::convert_pid_parameters(void)
 
     // make any SRV_Channel upgrades needed
     SRV_Channels::upgrade_parameters();
+
+    // convert AHRS parameters
+    AP::ahrs().convert_parameters();
```

Comment:
Yes, I agree but it's an existing issue (i.e. see existing SRV_Channels::upgrade_parameters() call).  I think I will leave this request to a future PR because it will lead to scope creep of an already large PR.